R/riskmetic.R

```diff
@@ -15,8 +15,12 @@ assessment <- function(assessment) {
 
   out <- as.list(assessment)
 
+  covr_coverage <- out[["covr_coverage"]]
+  if (length(covr_coverage) > 1) {
+    covr_coverage <- covr_coverage[["totalcoverage"]]
+  }
 
-  out[["covr_coverage"]] <- out[["covr_coverage"]][["totalcoverage"]]
+  out[["covr_coverage"]] <- covr_coverage
```

Comment:
Not sure what will happen when there is totalcoverage and then individual coverage for each file.

Reply:
It was not my intention to dive into this problem - I only made this quick fix to make sure I can generate a report. Otherwise it would crash.

This PR is merging my changes into your feature branch - so you can keep working on the reporting part itself.